content/200-concepts/200-database-connectors/08-cockroachdb.mdx

```diff
@@ -0,0 +1,71 @@
+| `xml`                         | `String`  |  Not yet  | `@db.Xml`                      | Supported in PostgreSQL but [not currently in CockroachDB](https://github.com/cockroachdb/cockroach/issues/43355)          |
+| `jsonb` arrays                | `Json[]`  |  Not yet  | N/A                            | `Json[]` supported in PostgreSQL but [not currently in CockroachDB](https://github.com/cockroachdb/cockroach/issues/23468) |
+
+## Other limitations
+
+The following table lists any other current known limitations of CockroachDB compared to PostgreSQL:
+
+| Issue                                                                | Area   | Notes                                                                                                                                                                                                                                                  |
+| -------------------------------------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `@@index(type: Hash)` not supported                                  | Schema | In PostgreSQL, Prisma allows [configuration of indexes](/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql) to use the `Hash` index access method. This is currently not supported in CockroachDB. |
+| Pushing to `Enum` types not supported                                | Client | Pushing to `Enum` types (e.g. `data: { enum { push: "A" }, }`) is currently [not supported in CockroachDB](https://github.com/cockroachdb/cockroach/issues/71388)                                                                                      |
+| Searching on `String` fields without a full text index not supported | Client | Searching on `String` fields without a full text index (e.g. `where: { text: { search: "cat & dog", }, },`) is currently [not supported in CockroachDB](https://github.com/cockroachdb/cockroach/issues/7821)                                          |
+| Integer division not supported                                       | Client | Integer division (e.g. `data: { int: { divide: 10, }, }`) is currently [not supported in CockroachDB](https://github.com/cockroachdb/cockroach/issues/41448)                                                                                           |
+| Limited filtering on `Json` fields                                   | Client | Currently CockroachDB [only supports](https://github.com/cockroachdb/cockroach/issues/49144) `equals` and `not` filtering on `Json` fields                                                                                                             |
```

Comment:
This adds two temporary limitations until CockroachDB release 22.1, see https://github.com/prisma/prisma/issues/11158#issuecomment-1018556344 and (internal) https://prisma-company.slack.com/archives/C01CSQXTNJ3/p1642775881029100
```suggestion
| Limited filtering on `Json` fields                                   | Client | Currently CockroachDB [only supports](https://github.com/cockroachdb/cockroach/issues/49144) `equals` and `not` filtering on `Json` fields                                                                                                             |
| Primary keys are named `primary` instead of `TABLE_pkey` (the Prisma default) causing them to be introspected as `@id(map: "primary")` | Introspection | [CockroachDB 22.1 will fix this](https://github.com/cockroachdb/cockroach/pull/70604)  |
| Foreign keys are named `fk_COLUMN_ref_TABLE` instead of `TABLE_COLUMN_fkey` (the Prisma default) causing them to be introspected as `@relation([...], map: "fk_COLUMN_ref_TABLE")` | Introspection | [CockroachDB 22.1 will fix this](https://github.com/cockroachdb/cockroach/pull/70658)                                                                                                             |
```
Might actually be more important as all the other "other limitations" as many more people will encounter these, so feel free to fix position and wording.

Reply:
I've added these in to the top of the table, and changed the wording a bit.